skyplane/api/dataplane.py

```diff
@@ -44,6 +44,9 @@ def __init__(
         transfer_config: TransferConfig,
     ):
         self.topology = topology
+        self.src_region_tag = self.topology.source_region()
+        self.dst_region_tag = self.topology.sink_region()
+        self.max_instances = int(len(self.topology.gateway_nodes) / 2)
```

Comment:
I don't think this is the case if there is an overlay.

Reply:
Can you compute the max instances across every region and use that instead?

Reply:
By the logic [here](https://github.com/skyplane-project/skyplane/blob/9e6c937fe4ec0aa0fd106d43ffc2ebd1000181fe/skyplane/replicate/replication_plan.py#L116-L126), it should be the same?

Yeah I can change that.

Reply:
What is this quantity used to report? Is it the max number of VMs per region used in the transfer? If so, consider if we send data via 2xA => 2xB => 2xC. We would get `max_instances=3`.

Reply:
We want `max_instances=2`. Oh I mean the src_region_tag and dst_region_tag should be the same. I already fix the max_instance issue, will push soon.